tests/test_reference_count_async.py

```diff
@@ -42,11 +41,8 @@ async def test_memory_objects() -> None:
 
     gc.collect()
 
-    df_dicts = pd.DataFrame()
-    df_dicts["dicts"] = objgraph.by_type("dict")
-    df_dicts["pw_types"] = df_dicts["dicts"].apply(lambda x: x.get("_type"))
+    pw_types = {d.get("_type") for d in objgraph.by_type("dict")}
```

Comment:
a PR should only contain a single change, please rebase to master.

Reply:
It was still a draft